frontend/.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true,
```

Comment:
This is not the only thing we'd need prettier to change in order to keep most of the rules we have.
Running prettier I see lots of changes with one especially that I'd like it to change: it's enforcing 80 chars but we've been enforcing 100.

Can you check whether this is possible? @ashu8912 had mentioned that prettier could reuse the eslint rules we've defined.